### PR TITLE
Main facility appointment phone number is not displayed when needed

### PR DESCRIPTION
**Pull request by brangi**
## Description
issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/18824

## Screenshots
Screen Shot 2021-01-27 at 12 17 59 PM

## Acceptance criteria
- [x]  Modify liquid template logic to display the main phone number (fieldPhoneNumber ) if fieldUseMainFacilityPhone is true AND fieldPhoneNumbersParagraph is empty.

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/health_care_local_facility.drupal.liquid [hunk 1]
```diff
@@ -187,6 +187,7 @@
                 {% include "src/site/facilities/facility_health_service.drupal.liquid" with
                     healthService = localHealthService
                     locationEntity = localService.entity
+                    mainFacilityPhoneNumber = fieldPhoneNumber
```
Comment (mpelzsherman) [inline src/site/layouts/health_care_local_facility.drupal.liquid:190]: I don't think we need this. We should already have the fieldPhoneNumber in the parent template. We might need to pass it down or `assign` it.

File: src/site/facilities/service_location.drupal.liquid [hunk 2]
```diff
@@ -101,6 +106,28 @@
...
+    {% endcomment %}
+
+    {% assign useMainFacilityPhone  = locationEntity.fieldServiceLocation.0.entity.fieldUseMainFacilityPhone  %}
+    {% if useMainFacilityPhone %}
+      <div class="vads-u-display--flex vads-u-flex-direction--column" data-template="paragraphs/service_location">
+        {% if haveLocationName %}
+          <h4 class="force-small-header">Phone</h4>
+        {% else %}
+          <h5>Phone</h5>
+          <a aria-label="{{ mainFacilityPhoneNumber | accessibleNumber }}"
```
Comment (mpelzsherman) [inline src/site/facilities/service_location.drupal.liquid:123]: Let's use the `fieldPhoneNumber` field here instead. We might need to pass it down from the parent, or assign it.

File: src/site/layouts/health_care_local_facility.drupal.liquid (hunk 1 above)
Comment (brangi) [reply src/site/layouts/health_care_local_facility.drupal.liquid:190]: isn't it exactly what I am doing `passing it` ?

Comment (mpelzsherman) [reply src/site/layouts/health_care_local_facility.drupal.liquid:190]: You added a new field to the GraphQL query that isn't needed.

File: src/site/facilities/service_location.drupal.liquid (hunk 2 above)
Comment (brangi) [reply src/site/facilities/service_location.drupal.liquid:123]: ok

**Review by mpelzsherman — APPROVED** [review]

**Pull request merged**